desktop/apps/auction2/components/filter_sort/index.js

```diff
@@ -10,6 +10,10 @@ function FilterSort(props) {
     updateSortAction
   } = props
   const selectedSort = filterParams.sort
+  const itemHeight = 37
```

Comment:
I think I could hook into the `componentDidMount` handler and update it there, but that seems a little messy with our current setup. Maybe let's leave it for now? (famous last words, I know 😬 )